db/PostgreSQL/seederPostgreORM.js

```diff
@@ -0,0 +1,115 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Let me know if your Sequelize implementation works. I'm curious to see how it turns out.